src/Bicep.Core.Samples/Bicep.Core.Samples.csproj

```diff
@@ -6,7 +6,11 @@
 
   <ItemGroup>
     <None Remove="./Files/**/*.*" />
-    <EmbeddedResource Include="./Files/**/*.*" LogicalName="$([System.String]::new('Files/%(RecursiveDir)%(Filename)%(Extension)').Replace('\', '/'))" />
+    <!-- 
+      .net 7 has a regression on Linux and Mac that assumes that if a file is named <name>.<something>.json that <something> is a culture
+      this causes some of our *.metadata.json files to be stripped out of the main assembly and placed in a satellite assembly for the metadata culture
+    -->
+    <EmbeddedResource Include="./Files/**/*.*" LogicalName="$([System.String]::new('Files/%(RecursiveDir)%(Filename)%(Extension)').Replace('\', '/'))" WithCulture="false" />
```

Comment:
We've followed this same pattern for embedding files in a number of test assemblies - should we fix them all?